src/index.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Nav from './components/Nav';
 
-const app = document.createElement('app');
-ReactDOM.render(<Nav />, app);
+
+ReactDOM.render(<Nav />, document.getElementById('app')|| document.createElement('div'));
 
 const sampleFunction = (a, b) => a + b;
 
```

Comment:
this is nice work mn. but this sample function we no longer needs so it should be deleted